### PR TITLE
Refactoring/status code

### PR DESCRIPTION
**Pull request by neewy**
## What is this pull request?

This pull request 
- updates cpp_redis and tacopie library up to 4.3.0 version with fixes
- changes transaction status name from ON_PROCESS to IN_PROGRESS
- fixes failing tests in Torii module
- adds removal of CLion debug folder in clean.sh
   
## Why do you implement it? Why do we need this pull request?
Initially I was planning only to change transaction status name. However, I had updated version of cpp_redis library, so I decided to update the project.

### CODE REVIEW

**Comment by neewy** [comment]
@Warchant please update cpp_redis lib in CI

**Comment by neewy** [comment]
Reason why builds are failing — Circle CI is using image which is not updated. Consider reviewing it on a local machine and after approval we will update image together with rebase

File: docker/develop/Dockerfile [hunk 1]
```diff
@@ -136,9 +136,9 @@ RUN git clone https://github.com/jtv/libpqxx /tmp/libpqxx; \
 
 # install cpp_redis with tacopie
 RUN git clone https://github.com/Cylix/cpp_redis /tmp/cpp_redis; \
-    (cd /tmp/cpp_redis ; git checkout 727aa5f06c8ce498168cbab5a023cad5b9c00bc0); \
+    (cd /tmp/cpp_redis ; git checkout f390eef447a62dcb6da288fb1e91f25f8a9b838c); \
```
Comment (Warchant) [inline docker/develop/Dockerfile]: for each  hash add comment with version

File: irohad/torii/torii_service_handler.cpp [hunk 2]
```diff
@@ -43,7 +43,14 @@ namespace torii {
   /**
    * shuts down service handler. (actually, shuts down completion queue only)
    */
-  void ToriiServiceHandler::shutdown() { completionQueue_->Shutdown(); }
+  void ToriiServiceHandler::shutdown() { completionQueue_->Shutdown();
```
Comment (Warchant) [inline irohad/torii/torii_service_handler.cpp]: Either use `clang-format` to format your code, or move this line with code to the next line manually.

File: docker/develop/Dockerfile (hunk 1 above)
Comment (neewy) [reply docker/develop/Dockerfile]: done

File: docker/develop/Dockerfile [hunk 3]
```diff
@@ -136,9 +136,9 @@ RUN git clone https://github.com/jtv/libpqxx /tmp/libpqxx; \
 
 # install cpp_redis with tacopie
 RUN git clone https://github.com/Cylix/cpp_redis /tmp/cpp_redis; \
-    (cd /tmp/cpp_redis ; git checkout f390eef447a62dcb6da288fb1e91f25f8a9b838c); \
+    (cd /tmp/cpp_redis ; git checkout f390eef447a62dcb6da288fb1e91f25f8a9b838c); \ # v4.3.0
```
Comment (Warchant) [inline docker/develop/Dockerfile]: it does not work like this.
`\` must be the very last token on the line

put comment before the line

Comment (neewy) [reply docker/develop/Dockerfile]: Done

**Comment by Warchant** [comment]
Don't forget to rebase.

**Comment by l4l** [comment]
ci doesn't like it :(

**Pull request closed without merging**